e2etests/test/server_valid.js

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+'use strict';
+
+var serviceSdk = require('azure-iothub');
+var serviceSas = require('azure-iothub').SharedAccessSignature;
+var provisioningServiceClient = require('azure-iot-provisioning-service').ProvisioningServiceClient;
```

Comment:
would much rather separate provisioning & iot hub tests.

Reply:
Ok.

---
In reply to: [220400315](https://github.com/Azure/azure-iot-sdk-node/pull/378#discussion_r220400315) [](ancestors = 220400315)